pnpm-wireit/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "homepage": "https://github.com/FormidableLabs/monorepo-infra-experiments#readme",
   "scripts": {
+    "version": "pnpm changeset version && pnpm install",
```

Comment:
What's `&& pnpm install` doing?

Reply:
Ah -- so the `changeset version` changes need an updated lockfile. I had CI fail for this reason and then googled and found https://github.com/changesets/changesets/issues/421

I do so wish that JSON allowed code comments 😛

Reply:
Actually, reading that more, I think I should be doing `pnpm install --lockfile-only`. I'll try that in next PR